scripts/build.py

```diff
@@ -825,7 +825,7 @@ def save_hashes_to_file(file_hashes):
     ensure_directory_exists(HASHES_TS_FILEPATH)
     with python_utils.open_file(HASHES_TS_FILEPATH, 'w+') as hashes_json_file:
         hashes_json_file.write(
-            unicode(json.dumps(filtered_hashes, ensure_ascii=False)))
+            python_utils.STR(json.dumps(filtered_hashes, ensure_ascii=False)))
```

Comment:
should I use convert_to_bytes here?

Reply:
I don't have a strong preference, use what you think that is more appropriate.